### PR TITLE
NewAzIpConfigurationBgpPeeringAddressObjectCommand should accept empty list of  Custom IP addresses

### PR DESCRIPTION
**Pull request by Khushboo-Baheti**
## Description

NewAzIpConfigurationBgpPeeringAddressObjectCommand should accept empty list of  Custom IP addresses
Updated validation check and help file

## Checklist

- [X] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Network/Network/help/Set-AzVirtualNetworkGateway.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ The first command gets a virtual network gateway named Gateway01 that belongs to
 The second command updates the virtual network gateway Gateway01 with the AAD authentication configurations params:aadTenantUri, aadAudienceId, aadIssuerUri for VpnClient.
 The third command removes the AAD authentication configuration from VpnClient of virtual network gateway.
 
-### Example 5: Add/Update IpConfigurationBgpPeeringAddresses to an existing virtual network gateway
+### Example 5.1: Add/Update IpConfigurationBgpPeeringAddresses to an existing virtual network gateway
```
Comment (VeryEarly) [inline src/Network/Network/help/Set-AzVirtualNetworkGateway.md:211]: Example 5

File: src/Network/Network/help/Set-AzVirtualNetworkGateway.md [hunk 2]
```diff
@@ -278,6 +278,74 @@ The third command assigns the address list into addresslist1.
 The fourth command created a PSIpConfigurationBgpPeeringAddress object.
 The fifth command set this new created PSIpConfigurationBgpPeeringAddress to IpConfigurationBgpPeeringAddresses and update the gateway.
 
+### Example 5.2: Update/Remove CustomAddress to an existing IpConfigurationBgpPeeringAddresses of virtual network gateway
```
Comment (VeryEarly) [inline src/Network/Network/help/Set-AzVirtualNetworkGateway.md:281]: Example 6

**Comment by VeryEarly** [comment]
please adjust example numbers, 5.1, 5.2 are not allowed

File: src/Network/Network/help/Set-AzVirtualNetworkGateway.md (hunk 1 above)
Comment (Khushboo-Baheti) [reply src/Network/Network/help/Set-AzVirtualNetworkGateway.md]: Updated to 5

File: src/Network/Network/help/Set-AzVirtualNetworkGateway.md (hunk 2 above)
Comment (Khushboo-Baheti) [reply src/Network/Network/help/Set-AzVirtualNetworkGateway.md]: Updated to 6

**Comment by Khushboo-Baheti** [comment]
> please adjust example numbers, 5.1, 5.2 are not allowed
Updated as per your suggestion. Thanks

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**